TW-1476 Doc Postbot November changes (#5207)
postmanlabs/postman-docs: src/pages/docs/getting-started/basics/about-postbot.md

Here are ways you can use Postbot:

> * **Write tests** - Postbot can add tests to your API requests based on your prompt. Use Postbot to add a new set of tests, visualize responses, save a field from a response, or fix your existing tests. For more information, see [using Postbot to write tests](/docs/writing-scripts/test-scripts/#using-postbot-to-write-tests).

Reviewer: ```suggestion
* **Write tests** - Postbot can add tests to your API requests based on your prompt. Use Postbot to add a new set of tests, visualize responses, save a field from a response, or fix your existing tests. For more information, see [using Postbot to write tests](/docs/writing-scripts/test-scripts/#write-tests-using-postbot).
```

* **Visualize responses** - Use Postbot to create a [Visualizer](/docs/sending-requests/visualizer/) to easily view your response data as a table, chart, or graph without writing any code.

> * **Write FQL in Postman Flows** - _Flows Query Language_ (FQL) enables you to parse and transform JSON data to get the fields and structure you want. You can ask Postbot to generate FQL for you. Enter an idea for a query in simple text, and Postbot will suggest an FQL query for your. For more information, see [Using Postbot with FQL](/docs/postman-flows/flows-query-language/introduction-to-fql/#using-postbot-with-fql)

Reviewer: ```suggestion
* **Write FQL in Postman Flows** - _Flows Query Language_ (FQL) enables you to parse and transform JSON data to get the fields and structure you want. You can ask Postbot to generate FQL for you. Enter an idea for a query in simple text, and Postbot will suggest an FQL query for you. For more information, see [Using Postbot with FQL](/docs/postman-flows/flows-query-language/introduction-to-fql/#using-postbot-with-fql)
```

> * **Write FQL in Postman Flows** - _Flows Query Language_ (FQL) enables you to parse and transform JSON data to get the fields and structure you want. You can ask Postbot to generate FQL for you. Enter an idea for a query in simple text, and Postbot will suggest an FQL query for your. For more information, see [Using Postbot with FQL](/docs/postman-flows/flows-query-language/introduction-to-fql/#using-postbot-with-fql)

Reviewer 2: ```suggestion
* **Write FQL in Postman Flows** - _Flows Query Language_ (FQL) enables you to parse and transform JSON data to get the fields and structure you want. You can ask Postbot to generate FQL queries for you. Enter an idea for a query in simple text, and Postbot will suggest an FQL query for your. For more information, see [Using Postbot with FQL](/docs/postman-flows/flows-query-language/introduction-to-fql/#using-postbot-with-fql)
```

> * **Write documentation** - Postbot can write documentation for your requests. Use Postbot to automatically add descriptions, including request parameters and responses examples to your requests. For more information, see [Writing documentation with Postbot](/docs/publishing-your-api/authoring-your-documentation/#writing-documentation-with-postbot).

Reviewer 2: ```suggestion
* **Write documentation** - Postbot can write documentation for your requests. Use Postbot to automatically add descriptions, including request parameters and response examples to your requests. For more information, see [Writing documentation with Postbot](/docs/publishing-your-api/authoring-your-documentation/#writing-documentation-with-postbot).
```

> * **Write documentation** - Postbot can write documentation for your requests. Use Postbot to automatically add descriptions, including request parameters and responses examples to your requests. For more information, see [Writing documentation with Postbot](/docs/publishing-your-api/authoring-your-documentation/#writing-documentation-with-postbot).

Reviewer 2: ```suggestion
* **Write documentation** - Postbot can write documentation for your requests. Use Postbot to automatically add descriptions, including request parameters and response examples to your requests. For more information, see [Writing documentation with Postbot](/docs/publishing-your-api/authoring-your-documentation/#writing-documentation-with-postbot).
```

> * **Get help** - Postbot is trained on Postman’s Learning Center documentation, too. Try asking Postbot for help if you get stuck — it will offer simple steps that you can follow and a link to more in-depth documentation.

Reviewer 2: ```suggestion
* **Get help** - Postbot is trained on Postman’s Learning Center documentation, too. Try asking Postbot for help if you get stuck — it will offer simple steps that you can follow and point to more in-depth documentation.
```
